src/test/scala/nodes/learning/LBFGSSuite.scala

```diff
@@ -0,0 +1,89 @@
+package nodes.learning
+
+import breeze.linalg._
+import edu.berkeley.cs.amplab.mlmatrix.RowPartitionedMatrix
+import org.apache.spark.SparkContext
+import org.scalatest.FunSuite
+import pipelines.{LocalSparkContext, Logging}
+import utils.{MatrixUtils, Stats}
+
+class LBFGSSuite extends FunSuite with LocalSparkContext with Logging {
+  test("Solve a dense linear system (fit intercept)") {
+    sc = new SparkContext("local", "test")
+
+    // Create the data.
+    val A = RowPartitionedMatrix.createRandom(sc, 128, 5, 4, cache=true)
+    val x = DenseMatrix((5.0, 4.0, 3.0, 2.0, -1.0), (3.0, -1.0, 2.0, -2.0, 1.0))
+    val b = A.mapPartitions(part => part * x.t)
```

Comment:
I think we should be adding an intercept to `A*x` to create the system here - especially if we want to check if the intercept is being fit